internal/cli/images.go

```diff
@@ -57,7 +60,14 @@ func runImagesCmd(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
+		err = bar.Add(1)
+		if err != nil {
+			return err
+		}
+	}
+	err = bar.Finish()
+	if err != nil {
+		return err
 	}
-
 	return nil
```

Comment:
Since this is the last error you can just return:

```go
return bar.Finish()
```

Reply:
I removed entirely the progress bar errors return, and just logged them, since I don't think it's ok to stop the program execution just because the UI bar failed to do something

Reply:
Sounds good. I've fixed a copy/paste but LGTM. Let's merge it! Thank you!